lib/ansible/modules/cloud/openstack/os_keystone_service_endpoint.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
```

Comment:
This file was not removed when renaming os_keystone_service_endpoint.py to os_keystone_service.py